pkg/scheduler/generic_scheduler.go

```diff
@@ -73,37 +65,30 @@ type ScheduleResult struct {
 	FeasibleNodes int
 }
 
-type genericScheduler struct {
```

Comment:
If we finally decided to remove this struct, I think this filename is not fit anymore.

Reply:
Yes, I am prepared to merge it into `scheduer.go` in next commit.

Removing the struct first makes code review easier.

Reply:
+1 just add another commit in this PR to merge `generic...` to `scheduler.go` so that there will be no interim commit in the code base.

Reply:
Prefer this to be another commit rather than another pr.